packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```diff
@@ -101,6 +119,25 @@ install -p -m 0644 nvidia.conf %{buildroot}%{_cross_tmpfilesdir}
 install -d %{buildroot}%{_cross_libdir}/modules-load.d
 install -p -m 0644 %{S:202} %{buildroot}%{_cross_libdir}/modules-load.d/nvidia-dependencies.conf
 
+install -p -m 0644 %{S:203} %{buildroot}%{_cross_unitdir}
+
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia
+install -p -m 0644 %{S:204} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/nvidia/fabricmanager.cfg
```

Comment:
Shouldn't these lines be after the comment in line 127? These are also part of Fabric Manager, right?

Reply:
changed as suggested.